File: wrapUpTasks/java/PrintLeapYearsBetween1900To2025.java
public class PrintLeapYearsBetween1900To2025{
    
    public static void main(String[] args){
        
        int startingYear = 1900;
        int endingYear = 2025;

        
        for(int year = startingYear; year <= endingYear; year++){
            
            if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)){
                System.out.print(year+" ");
                
            }
        }
        System.out.println();
        
    }
}

File: wrapUpTasks/javaScript/printLeapYearsBetween1900To2025.js

let startingYear = 1900;
let endingYear = 2025;


for(let year = startingYear; year <= endingYear; year++){
    
    if ((year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)){
        console.log(year+" ");
        
    }
}
console.log();

    